ocaml-docs-ci.opam.template

```diff
@@ -1,3 +1,3 @@
 pin-depends: [
-  ["obuilder-spec.dev" "git+https://github.com/ocurrent/obuilder#a2f4cec1774967a54808e6d5c9dfb6a4d77bc9c8"]
+  ["obuilder-spec.dev" "git+https://github.com/ocurrent/obuilder#8dbe2dd66b344198c9709a3432ed55ba8a5cb113"]
```

Comment:
This can point to https://github.com/ocaml/opam-repository/pull/22467 once it's released.